src/create-context.js

```diff
@@ -25,7 +25,7 @@ export function createContext(defaultValue, contextId) {
 
 				this.getChildContext = () => ctx;
 
-				this.shouldComponentUpdate = function (_props) {
+				this.shouldComponentUpdate = function(_props) {
```

Comment:
I didn't intend to change the formatting here but it seems as though the precommit hooks are doing that for me. I might have my setup incorrect, or maybe this file just needs these fixes with the config that exists?

Reply:
All good, the same happens on my end. I guess somehow a commit sneaked through one PR that didn't apply the prettier formatting from the pre-commit hook.

Reply:
Try reinstalling node_modules. We recently upgraded our prettier version so maybe that's the reason for the discrepancy? Either way, no biggie.

```
> npm ls prettier
preact@10.13.2 C:\code\github\preactjs\preact
└── prettier@2.8.6
```